sync/validation_queue.go

```diff
@@ -196,6 +196,10 @@ func (vq *blockQueue) removefromDepMaps(block types.Hash32, valid bool, doneBloc
 
 func (vq *blockQueue) addDependencies(jobId interface{}, blks []types.BlockID, finishCallback func(res bool) error) (bool, error) {
 	vq.Lock()
+	if _, ok := vq.callbacks[jobId]; ok {
```

Comment:
How is it relevant to this PR?
Where is a UT to test this new case?

Reply:
its a temporary mitigation to a race issue
it will be fixed more thoroughly in a different pr